src/server/game/Server/WorldSession.h

```diff
@@ -145,6 +145,7 @@ namespace WorldPackets
     namespace Pet
     {
         class DismissCritter;
+        class RequestPetInfo;
```

Comment:
Really doesn't need to word Pet on it, since clearly it's inside a Pet namespace.

Reply:
> Really doesn't need to word Pet on it, since clearly it's inside a Pet namespace.

this is the code style for the packet class names